### PR TITLE
Feature/webumenia 1504 user collection detail of favourites list

### PR DESCRIPTION
**Pull request by igor-kamil**
# Description

Screenshot 2020-12-20 at 17 07 10

Fixes [WEBUMENIA-1504](https://jira.sng.sk/browse/WEBUMENIA-1504)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:

- [ ] I have updated the [CHANGELOG](../CHANGELOG.md)
- [x] I have requested at least 1 reviewer for this PR
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: resources/js/components/user-collections/ClearButton.vue [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <template>
-    <button @click="clearAllItems">
+    <button class="btn btn-sm btn-default btn-outline sans ml-4 mb-2" @click="clearAllItems">
```
Comment (eronisko) [inline resources/js/components/user-collections/ClearButton.vue:2]: Fwiw, you could've put defined the CSS classes in Blade too.

File: resources/lang/cs/user-collection.php [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<?php
+
+return array(
+    'title'            => 'Oblíbené díla',
+    'empty'            => 'Momentálně nemáte uložené žádné díla',
+    'content-intro'    => 'Zde naleznete díla, které jste si uložili mezi oblíbené. <br>
+Díla jsou uloženy pouze dočasně ve vašem internetovém prohlížeči.',
+    'content-usage'    => 'Dílo ze seznamu odstraníte kliknutím na hvězdičku. <br>
```
Comment (eronisko) [inline resources/lang/cs/user-collection.php:8]: Isn't it _klepnutím_?

**Review by eronisko — APPROVED** [review]
Nice. Thanks for this and all the other PRs.

🚀

File: resources/js/components/user-collections/ClearButton.vue (hunk 1 above)
Comment (igor-kamil) [reply resources/js/components/user-collections/ClearButton.vue:2]: good to know

File: resources/lang/cs/user-collection.php (hunk 2 above)
Comment (igor-kamil) [reply resources/lang/cs/user-collection.php:8]: I have to say.. I don't know exactly. According the https://korektor.lingea.cz/ .. it should be ok:
Screenshot 2021-01-07 at 12 21 49

**Pull request merged**